stable/kong/templates/controller-deployment.yaml

```diff
@@ -0,0 +1,146 @@
+{{- if .Values.ingressController.enabled -}}
+apiVersion: apps/v1beta2
+kind: Deployment
+metadata:
+  name: "{{ template "kong.fullname" . }}-controller"
+  labels:
+    app: "{{ template "kong.name" . }}"
+    chart: "{{ .Chart.Name }}-{{ .Chart.Version }}"
+    release: "{{ .Release.Name }}"
+    heritage: "{{ .Release.Service }}"
+    component: "app"
```

Comment:
The Kong deployment has the same component label. You need a different one here.